template/helper/player.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * @package    DOCman
+ * @copyright   Copyright (C) 2011 - 2014 Timble CVBA (http://www.timble.net)
+ * @license     GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link        http://www.joomlatools.com
+ */
+
+class ComFilesTemplateHelperPlayer extends KTemplateHelperAbstract
+{
+    protected static $_SUPPORTED_FORMATS = array(
+        'audio' => array('aac', 'mp3', 'ogg', 'flac','x-flac', 'wave', 'wav', 'x-wav', 'x-pn-wav'),
+        'video' => array('mp4', 'webm', 'ogg')
+    );
+
+    protected static $_DECLARED = false;
+
+    protected function _initialize(KObjectConfig $config)
+    {
+        if (! self::$_DECLARED) {
+
+            $template = $this->getObject('com:files.view.plyr.html')
+                            ->getTemplate()
+                            ->addFilter('style')
+                            ->addFilter('script')
+                            ->loadFile('com:files.player.default.html');
+
+            print $template->render();
```

Comment:
@ercanozkaya The reason I didn't make it a display method is because it isn't displaying anything. It is importing the js and css in html header and initializing the UI. That's why it is part of the _initialize method. Also it is not an optional method. The UI needs to be initiated at least once and only once, that's why it is automatically called rather than us calling it every time the behaviour is used? That said, I'd still change it if that's what you want.